float64 cumulative_hyperparam # average of estimates
float64 current_estimate # current estimate
int16 count # count of estimates up to now

float64 average_led_per_press
float64 last_led_per_press

float64 average_risk_taken
float64 last_risk_taken

float64 expertise
float64 last_expertise